src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -1,42 +1,50 @@
 package com.openlattice.chronicle;
 
 import com.google.common.base.Optional;
+import com.google.common.collect.SetMultimap;
+import com.openlattice.chronicle.data.FileType;
 import com.openlattice.chronicle.sources.Datasource;
-import java.util.UUID;
 import retrofit2.http.Body;
 import retrofit2.http.GET;
 import retrofit2.http.POST;
 import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+import java.util.UUID;
 
 public interface ChronicleStudyApi {
 
     String SERVICE    = "/chronicle";
     String CONTROLLER = "/study";
     String BASE       = SERVICE + CONTROLLER;
 
-    String STUDY_ID       = "studyId";
-    String PARTICIPANT_ID = "participantId";
     String DATASOURCE_ID  = "datasourceId";
+    String ENTITY_ID      = "entityId";
     String ENTITY_SET_ID  = "entitySetId";
+    String FILE_TYPE      = "fileType";
+    String PARTICIPANT_ID = "participantId";
+    String STUDY_ID       = "studyId";
+
+    String DATA_PATH        = "/data";
+    String PARTICIPANT_PATH = "/participant";
 
-    String STUDY_ID_PATH       = "/{" + STUDY_ID + "}";
-    String PARTICIPANT_ID_PATH = "/{" + PARTICIPANT_ID + "}";
     String DATASOURCE_ID_PATH  = "/{" + DATASOURCE_ID + "}";
+    String ENTITY_ID_PATH      = "/{" + ENTITY_ID + "}";
     String ENTITY_SET_ID_PATH  = "/{" + ENTITY_SET_ID + "}";
-
+    String PARTICIPANT_ID_PATH = "/{" + PARTICIPANT_ID + "}";
+    String STUDY_ID_PATH       = "/{" + STUDY_ID + "}";
 
     /**
      * Enrolls a participant's data datasource in a study. Currently the only supported datasource is an Android device, though
      * though that may change in the future.
-     *
+     * <p>
```

Comment:
is this a javadoc thing?

Reply:
Yes. It is being inserted automatically by the IntelliJ formatter.